### PR TITLE
Postman api 

### LINKED ISSUE #182: Create an API definition file
**Is this related to a problem? Please describe.**

Follow steps in postman. Create a definition that follows the OpenAPI data format.

### PR DESCRIPTION
**Pull request by edouarde1**
Created an OpenApi template, needs more work.

Hopefully this can stay a draft, but for gitstats stake it can be merged!

### CODE REVIEW

**Comment by vnajack** [comment]
For GitStats 🤺
We'll work on this and other issues next week 🙂

**Review by vnajack — APPROVED** [review]

File: postman/schemas/index.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          content:
+            video/mp4:
+              schema:
+                $ref: '#/components/schemas/AnimationFile'
+        404:
+          description: No animation file resource exists
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
```
Comment (LoganParker) [inline postman/schemas/index.yaml:37]:
```suggestion
        404:
          description: Animation files not found
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Error'
```

File: postman/schemas/index.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      properties:
+        file-contents:
+          description: .txt file upload
+    Recognizer:
+      type: string 
+      required:
+        - name
+      properties:
+        name:
+          description: a selection of one of the speech Recognizer's for Rhubarb
```
Comment (LoganParker) [inline postman/schemas/index.yaml:68]: We should add a field for character selection

File: postman/schemas/index.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      type: object
+      format: binary      
+      required:
+        - file-contents
+      properties:
+        file-contents:
+          description: .txt file upload
+    Recognizer:
+      type: string 
+      required:
```
Comment (LoganParker) [inline postman/schemas/index.yaml:64]: recognizer & text files I dont think are required

**Comment by LoganParker** [comment]
This looks really good! Defining the schema definitely is a hefty task so thank you!

**Review by LoganParker — APPROVED** [review]

**Review by mkuelker — APPROVED** [review]
Ready to merge!

**Pull request merged**